db_service/vcap_test.go

```diff
@@ -0,0 +1,287 @@
+			ExpectedError: nil,
+		},
+		"valid pivotal vcap service": {
+			VcapService:   VcapService{BindingName: "", InstanceName: "", Name: "my-instance", Label: "p.mysql", Tags: []string{"mysql"}, Plan: "db-medium", Credentials: map[string]interface{}{"hostname": "10.0.0.20", "jdbcUrl": "jdbc:mysql://10.0.0.20:3306/service_instance_db?user=fefcbe8360854a18a7994b870e7b0bf5&password=z9z6eskdbs1rhtxt", "name": "service_instance_db", "password": "z9z6eskdbs1rhtxt", "port": 3306, "uri": "mysql://fefcbe8360854a18a7994b870e7b0bf5:z9z6eskdbs1rhtxt@10.0.0.20:3306/service_instance_db?reconnect=true", "username": "fefcbe8360854a18a7994b870e7b0bf5"}},
```

Comment:
Would you mind formatting these kind of like the JSON above with line breaks? That would help readability long-term.


```
"valid pivotal vcap service": {
  VcapService:   VcapService{
    BindingName: "",
    InstanceName: "",
    Name: "my-instance",
    Label: "p.mysql",
    Tags: []string{"mysql"},
    Plan: "db-medium",
    Credentials: map[string]interface{}{
      "hostname": "10.0.0.20",
      "jdbcUrl": "jdbc:mysql://10.0.0.20:3306/service_instance_db?user=fefcbe8360854a18a7994b870e7b0bf5&password=z9z6eskdbs1rhtxt", 
      "name": "service_instance_db", 
      "password": "z9z6eskdbs1rhtxt", 
```